src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (number) => {
+    if (number) {
```

Comment:
```suggestion
    if (number === undefiend) {
```

Reply:
Agreed, it will not work correctly if you call `adder(0)`.